srv/SpawnEntity.srv

```diff
@@ -0,0 +1,19 @@
+# Spawn an entity (a robot, other object) by name or URI
```

Comment:
Where does "respawning an entity" come in -- a separate service?

I'm thinking for example, if you've spawned an apple on a table but you want to randomize its position without resetting the world?

Reply:
With the current interface, it would mean using SetEntityState on this spawned entity with a randomized pose.
Would this be sufficient? I am open to suggestions on including randomization in the interface if it is justified.

Reply:
You're right that it would be doable in this way, so probably not worth adding something new for this. Good call.